hyp3_autorift/vend/README.md

```diff
@@ -29,7 +29,7 @@ We've replaced it  with `hyp3_autorift.io.get_topsinsar_config`.
 ## Additional Patches
 
 1. The changes listed in `CHANGES-176.diff` were applied in [ASFHyP3/hyp3-autorift#176](https://github.com/ASFHyP3/hyp3-autorift/pull/176)
```

Comment:
A file named `CHANGES-176.diff` including changes beyond PR #176 annoys me somewhat; would just `CHANGES.diff` be a better name? Not a blocking concern.

Reply:
Agree but I don't like any other options better... so, meh, keeping it.